solidity-plugin/src/test/java/org/sonarsource/solidity/SoliditySensorTest.java

```diff
@@ -0,0 +1,5 @@
+package org.sonarsource.solidity;
+
+public class SoliditySensorTest {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add some tests to this class. ([squid:S2187](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3AS2187))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=11&issues=AWPzDwcc3e-qcckj23qy&open=AWPzDwcc3e-qcckj23qy)

[ISSUE_KEY]: AWPzDwcc3e-qcckj23qy